charts/prometheus-druid-exporter/templates/tests/connection-test.yaml

```
@@ -15,5 +15,6 @@ spec:
   - name: wget
     image: busybox
     command: ['wget']
-    args:  ['-qO-', '{{ include "prometheus-druid-exporter.fullname" . }}:{{ .Values.exporterPort }}/metrics']
+    args:  ['-qO-', '{{ include "prometheus-druid-exporter.fullname" . }}.svc:{{ .Values.exporterPort }}/metrics']
```

When dns is used the namesapce needs to be given too.
See: https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#services

So i guess it should look like:

```suggestion
    args:  ['-qO-', '{{ include "prometheus-druid-exporter.fullname" . }}.{{ .Release.Namespace }}.svc:{{ .Values.exporterPort }}/metrics']
```

---

Updated and pushed. Waiting for someone for pipeline to run.

---

hi @JulioZanette, I re-ran the CI and all your changes are working fine. I cannot merge the PR. 

You gotta check with @iamabhishek-dubey or @sandy724 to get this merged.